pkg/cachemanager/cachemanager_test/cachemanager_integration_test.go

```diff
@@ -35,92 +35,137 @@ const (
 
 var cfg *rest.Config
 
+var (
+	configMapGVK = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "ConfigMap"}
+	podGVK       = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"}
+
+	configInstancOne      = "config-test-1"
```

Comment:
typo: Instanc

Definitely nit but IMO `config1` > `configInstanceOne`.  Less to look at and just as meaningful.

Reply:
Similarly for all mentions of "instance"

Reply:
reanmed to `cm1Name` per Julian's suggestion.